packages/cli/src/databases/entities/folder.ts

```diff
@@ -29,6 +29,8 @@ export class Folder extends WithTimestampsAndStringId {
 	@OneToMany('WorkflowEntity', 'parentFolder')
 	workflows: WorkflowEntity[];
 
+	workflowsCount?: number;
```

Comment:
Since this is not in the DB, let's not add it to the entity either. Instead let's use a new interface that extends the entity and has this